src/ontology/fbbt-edit.obo

```diff
@@ -9696,11 +9696,14 @@ is_a: FBbt:00001040 ! embryonic/larval alary muscle cell
 
 [Term]
 id: FBbt:00001048
-name: larval heart circular muscle cell
-def: "Muscle cell of the circumscribing muscle on the outside of the embryonic/larval heart." [FlyBase:FBrf0031012]
+name: embryonic/larval contractile cardiomyocyte
+def: "Contractile cardiomyocyte of the embryonic/larval dorsal vessel. There are 76 of these cells. Their myofilaments are not arranged as regularly as in their adult counterparts, and may in some cells deviate from a strictly circular configuration (Lehmacher et al., 2012)." [FlyBase:FBrf0031012]
 subset: cur
 synonym: "circular muscle of larval heart" EXACT []
+synonym: "larval heart circular muscle cell" EXACT [FlyBase:FBrf0219200]
 is_a: FBbt:00001036 ! embryonic/larval heart visceral muscle cell
+intersection_of: FBbt:00058241 ! contractile cardiomyocyte
+intersection_of: part_of FBbt:00001673 ! embryonic/larval dorsal vessel
```

Comment:
For this and several other logical defs you have made here, the necessary and sufficient definition would be following the pattern: "'contractile cardiomyocyte' and 'part of' some larva", then you can add SubClassOf "'part of' some 'embryonic/larval dorsal vessel'".